src/frontend/src/lib/utils/UI/backgrounds/waveBackground.ts

```diff
@@ -315,6 +316,63 @@ export async function createImpulse(
   await Promise.all(promises);
 }
 
+export async function createOneWayImpulse(
```

Comment:
Could you add a comment to say how this is similar / different from createImpulse